app/bundles/FormBundle/Views/Field/freehtml.html.php

```diff
@@ -12,7 +12,7 @@
 include __DIR__.'/field_helper.php';
 
 if ($inBuilder) {
-    $htmlContent = $view['content']->showScriptTags($properties['text']);
```

Comment:
This was removed from my commits or from yours? 
This should prevent run script code from builder
We could revert it

![image](https://user-images.githubusercontent.com/462477/84110954-2a194600-aa26-11ea-9091-7e700476611b.png)

Reply:
Good catch. I will push a fix shortly together with other bug we found.

Reply:
To answer your question, this is the commit that removed it: https://github.com/mautic/mautic/commit/1293c0f470b769d4c6a9b8f1a9cc364f96688de6#diff-0ffe1ecc742a8abb035d9e423721e222R15

Reply:
Yes, I found it. But didn't have time check it why I remove it. Maybe accidentaly

Reply:
Fixed